crowbar_framework/config/locales/crowbar/en.yml

```diff
@@ -138,7 +142,7 @@ en:
       confirm_delete_proposal: 'Are you sure you wish to delete this proposal?'
       available_clusters: 'Available Clusters'
       available_clusters_hint: 'Here you can see all available clusters.'
-      available_remotes: 'Available Remotes'
+      available_remotes: 'Available Clusters with Remote Nodes'
```

Comment:
nitpick that is clearly for a different PR: we use title and lower case sentences a bit random